frontend/src/core/components/Dashboard/LoginStatus.js

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from "react";
+import {Dropdown} from "react-bootstrap";
+import PropTypes from "prop-types";
+
+
+// eslint-disable-next-line react/display-name
+const LabelToggle = React.forwardRef(({children, onClick}, ref) => (
+  <React.Fragment>
```

Comment:
Same comment about **Fragment**

Reply:
As here I'm using two sibling elements, should I replace it with a div?

Reply:
yes, according to this pattern: https://reactjs.org/docs/fragments.html